compute/computer.go

```diff
@@ -116,6 +119,34 @@ func (c *Computer) ExecuteTransaction(
 	}
 
 	delta := view.Delta()
+	prog := programs.NewEmptyPrograms()
+
+	states := extractStateChangesFromDelta(delta, func(address common.Address, key string) (value cadence.Value, err error) {
+		pathParts := strings.Split(key, "\x1F")
+
+		if len(pathParts) != 2 {
+			// not a cadence path value
+			return nil, nil
+		}
+
+		path := cadence.Path{
+			Domain:     pathParts[0],
+			Identifier: pathParts[1],
+		}
+
+		defer func() {
+			if r := recover(); r != nil {
+				value = nil
+				err = nil
+			}
+		}()
+
+		value, err = c.vm.Runtime.ReadStored(address, path, runtime.Context{Interface: &apiEnv{
+			Delta:    &delta,
+			Programs: prog,
+		}})
```

Comment:
Nice! Maybe extract the context outside, so it doesn't have to be recreated on each function call